TicketBooking.Data/Infrastructure/IRepository.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TicketBooking.Data.Infrastructure
+{
+    public interface IRepository<T,X> where T : class where X : class
+    {
```

Comment:
can we add more methods for included and order result?

Reply:
Sorry we do not get your question. You mean we should add methods that could get results in order?

Reply:
Yes. the method support ordering the result. Load relationships model.